src/account/update.rs

```diff
@@ -274,6 +274,13 @@ impl AccountHandle {
         for address in &mut account.internal_addresses {
             address.address.bech32_hrp = bech32_hrp.clone();
         }
+
+        #[cfg(feature = "storage")]
+        {
+            log::debug!("[SYNC] storing account {} with new synced data", account.alias());
```

Comment:
We didn't sync there, so maybe something like
```suggestion
            log::debug!("[SYNC] storing account {} after updating it with new client options", account.alias());
```

Reply:
should be fine like this now